src/main/java/edu/kis/powp/jobs2d/drivers/SelectDriverMenuOptionListener.java

```diff
@@ -7,17 +7,17 @@
 import edu.kis.powp.jobs2d.features.DriverFeature;
 
 public class SelectDriverMenuOptionListener implements ActionListener {
-	private DriverManager driverManager;
-	private Job2dDriver driver = null;
 
-	public SelectDriverMenuOptionListener(Job2dDriver driver, DriverManager driverManager) {
-		this.driverManager = driverManager;
-		this.driver = driver;
-	}
+    private DriverManager driverManager;
+    private Job2dDriver driver = null;
 
-	@Override
-	public void actionPerformed(ActionEvent e) {
-		driverManager.setCurrentDriver(driver);
-		DriverFeature.updateDriverInfo();
-	}
+    public SelectDriverMenuOptionListener(Job2dDriver driver, DriverManager driverManager) {
+        this.driverManager = driverManager;
+        this.driver = driver;
+    }
+
+    @Override public void actionPerformed(ActionEvent e) {
+        driverManager.setCurrentDriver(driver);
+        DriverFeature.updateDriverInfo();
```

Comment:
You should follow the project convention (i.e. tab size). Usually, such commits are rejected at code review phase.